### PR TITLE
starter viability model

### PR DESCRIPTION
**Pull request by GaryPWhite**
Per our conversations in a few -wg's, I've drafted a Viability Starter model. Happy to take comments and feedback right on the PR, Thanks!

### CODE REVIEW

File: metrics-model-libs/viability/starter/definition/viability-starter.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+## Insights Drawn from Metrics Model
+
+Context before insights: “Viability” is a term that will have different thresholds for risk tolerances across organizations and industries. The Compliance and Security that does not suit a telecommunications company may be suitable for a startup emerging into a new space with limited resources and libraries that address their problems. 
+
+Intended use of this metrics model is to feed into an overall viability determination. Every company, like with risk tolerance, will have different degrees of “viable” based on how the software is ultimately applied in a product/service.
+
+
+
+* [Bus Factor](https://chaoss.community/?p=3944)
+    * An low score on bus factor may indicate that a project is not durably maintained.. Smaller projects/organizations may feel more comfortable with a bus factor of 2, while larger organizations would expect bus factors to be much higher before they consider projects to be viable.
```
Comment (ElizabethN) [inline metrics-model-libs/viability/starter/definition/viability-starter.md:54]: fix this horrible double period, please.

Comment (ElizabethN) [inline metrics-model-libs/viability/starter/definition/viability-starter.md:54]:
```suggestion
    * An low score on bus factor may indicate that a project is not durably maintained. Smaller projects/organizations may feel more comfortable with a bus factor of 2, while larger organizations would expect bus factors to be much higher before they consider projects to be viable.
```

**Review by ElizabethN — APPROVED** [review]

**Comment by geekygirldawn** [comment]
@ElizabethN Do you know when this will go live on the website?

**Comment by ElizabethN** [comment]
@geekygirldawn I have an [open issue](https://github.com/chaoss/wg-metrics-models/issues/109) to remind me - it's on my list 😄 Will get it published today 🤞

**Pull request merged**